mackerel-plugin-memcached/memcached.go

```diff
@@ -147,7 +148,8 @@ func (m MemcachedPlugin) GraphDefinition() map[string](mp.Graphs) {
 			Label: (labelPrefix + " Items"),
 			Unit:  "integer",
 			Metrics: [](mp.Metrics){
-				mp.Metrics{Name: "curr_items", Label: "Current Items", Diff: false},
+				mp.Metrics{Name: "curr_items", Label: "Current Items", Diff: false, Type: "uint64"},
+				mp.Metrics{Name: "new_items", Label: "Current Items", Diff: true, Type: "uint64"},
```

Comment:
[memo]
Since mackerel-plugin-helper normalizes diff metric value using `diffTime`, this metric value will be float64, even if `Type: "uint64"` specified.
https://github.com/mackerelio/go-mackerel-plugin-helper/blob/master/mackerel-plugin.go#L179